src/components/sub-menu/SubMenu.vue

```diff
@@ -1,48 +1,46 @@
 <template>
   <v-menu
     :close-on-content-click="false"
-    :offset-x="isOffsetX"
-    :offset-y="isOffsetY"
     :open-on-hover="isOpenOnHover"
     :transition="transition"
-    :value="openMenu"
+    :model-value="openMenu"
   >
-    <template #activator="{ on }">
+    <template #activator="{ props }">
       <v-btn
         v-if="icon"
         :color="color"
-        v-on="on"
+        v-bind="props"
       >
         <v-icon>{{ icon }}</v-icon>
       </v-btn>
       <v-list-item
         v-else-if="isSubMenu"
         class="d-flex justify-space-between"
-        v-on="on"
+        append-icon="mdi-chevron-right"
+        v-bind="props"
       >
         {{ name }}
-        <div class="flex-grow-1" />
-        <v-icon>mdi-chevron-right</v-icon>
       </v-list-item>
       <v-btn
         v-else
         :color="color"
-        text
-        v-on="on"
+        variant="text"
+        v-bind="props"
         @click="openMenu=true"
       >
         {{ name }}
       </v-btn>
     </template>
     <v-list>
-      <template v-for="(item, index) in menuItems">
+      <template
+        v-for="(item, index) in menuItems"
+        :key="index"
```

Comment:
`:key` should be on template tag in Vue 3, not on the children: https://v3-migration.vuejs.org/breaking-changes/key-attribute.html